satellite/satellitedb/migrate_postgres_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/internal/dbutil/dbschema"
+	"storj.io/storj/internal/dbutil/pgutil"
+	"storj.io/storj/satellite/satellitedb"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+// VersionSchema defines a versioned schema
+type VersionSchema struct {
```

Comment:
This could be moved to `dbschema` package.

Reply:
Will pull into a separate PR.

Reply:
https://github.com/storj/storj/pull/1312